tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -703,27 +703,31 @@ class MklFusedBatchNormOp : public OpKernel {
   void Compute(OpKernelContext* context) override {
     try {
       auto cpu_engine = engine(engine::cpu, 0);
-      const size_t src_index = 0;    // index of src input tensor
-      const size_t scale_index = 1;  // index of scale tensor
-      const size_t shift_index = 2;  // index of shift tensor
-      const size_t mean_index = 3;   // index of est_mean tensor
-      const size_t var_index = 4;    // index of est_variance tensor
-
-      const Tensor& src_tensor          = MklGetInput(context, src_index);
-      const Tensor& scale_tensor        = MklGetInput(context, scale_index);
-      const Tensor& shift_tensor        = MklGetInput(context, shift_index);
-      const Tensor& est_mean_tensor     = MklGetInput(context, mean_index);
-      const Tensor& est_variance_tensor = MklGetInput(context, var_index);
-
+      const size_t kSrcIndex = 0;       // index of src input tensor
```

Comment:
Just curious, why rename the constants of type `size_t` while keeping Tensors unchanged? In the code guideline, such naming convention is optional for local non-static variables.

Reply:
To me this makes sense, as any variable that starts with "k" is a constant. And here, they are really constants, their values will never ever change. Almost like macros.
When I look at the tensors below, they are constants but their values will be set at runtime.

So IMHO this makes the code more readable for me.